io/teca_cf_layout_manager.cxx

```diff
@@ -15,6 +15,79 @@
 #include <cerrno>
 #include <string>
 
+namespace
+{
+    int full_path_varname_to_group_id_and_name_in_group(int file_id,
+        const std::string& full_path_varname, int &parent_id,
+        std::string& name_in_group)
+    {
+        std::cout << "full_path_varname_to_group_id_and_name_in_group: " << file_id << " " << full_path_varname;
```

Comment:
I assume the `std::cout` lines in this subroutine are debugging artefacts?  These should probably be removed or changed to only write if the `verbose` flag is set for the writer

Reply:
I already removed this but it seems I forgot to commit that change. Planning on just removing them.

Reply:
@taobrienlbl , I had removed that and performed a bit more clean-up but forgotten to commit. Can you please take another close look at the change, please?

Reply:
I see -- it looks like github is for some reason not showing me the most up-to-date version of this file.  I'm looking at the diff in visual studio code and see that you've removed this.